.github/workflows/cicd.yaml

```diff
@@ -28,61 +30,83 @@ jobs:
           registry: ghcr.io
           username: ${{ github.actor }}
           password: ${{ secrets.GH_PAT }}
+          logout: false
 
 
-      - name: Setup Custom libvirt RPMs
+      - name: Setup Custom libvirt & qemu RPMs
         run: |
-          echo "Setting up custom libvirt RPMs from pre-built image..."
-          
-          # Pull the pre-built RPMs container
+          set -euo pipefail
+          echo "Setting up custom libvirt & qemu RPM repos..."
+
+          # Pull pre-built RPM images
           docker pull ${{ env.CUSTOM_LIBVIRT_IMAGE }}
+          docker pull ${{ env.CUSTOM_QEMU_IMAGE }}
+
+          docker rm -f libvirt-rpms-http-server 2>/dev/null || true
+          docker rm -f qemu-rpms-http-server 2>/dev/null || true
           
-          if [ "$(docker ps -q -f name=rpms-http-server)" ]; then
-            echo "RPMs HTTP server is already running."
-          else
-            echo "Starting RPMs HTTP server..." 
+          #TODO: Remove this line after verifying both repos are accessible 
```

Comment:
This is not needed now that we have separate http servers for qemu and libvirt, right?